ajax/libs/videojs-ima/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "videojs-ima",
+  "filename": "videojs.ima.min.js",
+  "description": "IMA SDK Plugin for Video.js ",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Google Inc."
+  },
```

Comment:
I think using `"author": "Google Inc."` is better.
because it just need fill in the author name.